### PR TITLE
[MB-13792] EvaluationReport subcomponents rename

### PR DESCRIPTION
**Pull request by SirenaBorracha**
## Summary

As TTV built further on the EvaluationReport components, the naming conventions became confusing. This story updates component names to improve comprehensibility for current and future work.

There are no visual changes in the FE, though Storybook titles have been updated which caused the Happo diffs.

## Changes and recommendations

`EvaluationReportShipmentInfo` is now `ShipmentQAEReportHeader`
	src/components/Office/EvaluationReportTable/EvaluationReportShipmentInfo.jsx

`EvaluationViolationsForm` is now `QAEViolationsForm`
	src/components/Office/EvaluationViolationsForm/EvaluationViolationsForm.jsx

`EvaluationReportTable` is now `QAEReportTable`
	src/components/Office/EvaluationReportTable/EvaluationReportTable.jsx

`ShipmentEvaluationReports` is now `ShipmentQAEReports`
	src/components/Office/EvaluationReportTable/ShipmentEvaluationReports.jsx

Next I would recommend refactoring `QAEReportTable` to create `CounselingQAEReports` matching what was already done for `ShipmentQAEReports`. I think it would help with comprehension, that or leave a comment that `QAEReportTable` does the work for Counseling reports but imports the logic for Shipment reports. I'm open to either as part of this PR.

## Setup to Run Your Code

<details>
<summary>💻 You will need to use three separate terminals to test this locally.</summary>

##### Terminal 1

Start the Storybook locally.

```sh
make storybook
```

##### Terminal 2

Start the UI locally.

```sh
make client_run
```

##### Terminal 3

Start the Go server locally.

```sh
make server_run
```

</details>

### Additional steps

1. Access Storybook and search for each of the four renamed components
2. Separately, login as a QAE/CSR and navigate to a move, then to the reports tab to confirm that the reports components behave as expected

Move codes and their respective move types
* `REQSRV`     NTS-r
* `R3TNTR`     NTS-r
* `JRJB8D`      HHG
* `NTSEVR`     NTS-r && NTS

## Verification Steps for Author

These are to be checked by the author.

- [n/a] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [x] Request review from a member of a different team.
- [x] Have the Jira acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Frontend

- [ ] User facing changes have been reviewed by design.
- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/main/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
- [ ] There are no new console errors in the browser devtools
- [ ] There are no new console errors in the test output
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.

### Backend

- [ ] Code follows the guidelines for [Logging](https://transcom.github.io/mymove-docs/docs/dev/contributing/backend/Backend-Programming-Guide#logging)
- [ ] The requirements listed in [Querying the Database Safely](https://transcom.github.io/mymove-docs/docs/dev/contributing/backend/Backend-Programming-Guide/#querying-the-database-safely) have been satisfied.

#### Any new migrations/schema changes:

- [ ] Follows our guidelines for [Zero-Downtime Deploys](https://transcom.github.io/mymove-docs/docs/dev/contributing/database/Database-Migrations#zero-downtime-migrations)
- [ ] Have been communicated to #g-database
- [ ] Secure migrations have been tested following the instructions in our [docs](https://transcom.github.io/mymove-docs/docs/dev/contributing/database/Database-Migrations#secure-migrations)

### CODE REVIEW

**Review by mandaem — COMMENTED** [review]
Screenshot 2022-11-22 at 1 00 55 PM
There are some styling issues on the reports page.

File: src/components/Office/QAEReportTable/QAEReportTable.module.scss [hunk 1]
```diff
@@ -2,7 +2,7 @@
 @import 'shared/styles/_mixins';
 @import 'shared/styles/colors';
 
-.evaluationReportTable {
```
Comment (PDickeyTruss) [inline src/components/Office/QAEReportTable/QAEReportTable.module.scss:5]: Looks like this is the line that is causing the styling issues. In the QAEReportTable component this is still 'evaluationReportTable' so this style is not getting used.

Comment (PDickeyTruss) [reply src/components/Office/QAEReportTable/QAEReportTable.module.scss:5]: See line 167 of QAEReportTable for the mismatch: `<table className={styles.evaluationReportTable}>`

**Review by PDickeyTruss — CHANGES_REQUESTED** [review]
Looking good, very nice naming cleanup. Makes it a bit easier to navigate the QAE fiels than before this PR.

Marking as 'Request changes' for the small 1-line styling fix. On standby to approve when fixed 🎉

**Review by PDickeyTruss — APPROVED** [review]
LGTM! Styling fixed and renaming still solid 🎉

**Review by dinabeigelman — APPROVED** [review]
Assuming there are no visual/UI changes - this looks good!

**Pull request merged**